### PR TITLE
Support testing nextcloud-spreed-signaling from Debian 12~bpo or Debian 13

### PR DESCRIPTION
**Pull request by sunweaver**
Depends on #127

### CODE REVIEW

**Comment by sunweaver** [comment]
This PR now adds general Debian 13 support + supports testing nextcloud-spreed-signaling from Debian bookworm-backports.

File: setup-nextcloud-hpb.sh [hunk 1]
```diff
@@ -301,19 +302,42 @@ function show_dialogs() {
...
+					13 65 3>&1 1>&2 2>&3; then
+					SIGNALING_BUILD_FROM_SOURCES=true
+				fi
+			fi
 		else
-			whiptail --title "Building from sources." \
+			# Originally, this part was for running this script on Debian 10 which is not
+			# supported anymore.
+			#
+			# Nowadys, most probable reason for landing here is if people try this script
```
Comment (dzatoah) [inline setup-nextcloud-hpb.sh:331]:
```suggestion
			# Nowadays, most probable reason for landing here is if people try this script
```

File: src/setup-signaling.sh [hunk 2]
```diff
@@ -26,6 +26,15 @@ declare -A SIGNALING_NC_SERVER_MAXSCREENBITRATE # Associative array
 function install_signaling() {
 	log "Installing Signaling…"
 
+	if [ "$DEBIAN_VERSION_MAJOR" = "12" ] ; then
+		log "Enable bookworm-backports"
+		is_dry_run || cat <<EOL >$SIGNALING_BACKPORTS_SOURCE_FILE
```
Comment (dzatoah) [inline src/setup-signaling.sh:31]: Please use `cat <<-EOL` and indent the following lines accordingly.

**Review by dzatoah — COMMENTED** [review]
Looks good! 2 papercuts, tho. I'll fix them right away.

**Comment by dzatoah** [comment]
@sunweaver merge if you want to. I approve. :+1:

**Pull request merged**